spec/oncocore.txt

```diff
@@ -199,13 +204,13 @@ Based on:		SurgicalProcedure
 Description:	"A surgical action addressing a cancer condition, independent of action context."
 // It is cancer-related only if the reason is a cancer condition
 0..1			ReasonReference value is type CancerCondition
-0..1			ReasonCode from CancerDisorderVS
+0..1			ReasonCode from AnyCancerDisorderCodeVS
 
 EntryElement:	CancerRelatedRadiationProcedure
 Based on:		RadiationProcedure
 // It is cancer-related only if the reason is a cancer conditio
 0..1			ReasonReference value is type CancerCondition
-0..1			ReasonCode from CancerDisorderVS
+0..1			ReasonCode from AnyCancerDisorderCodeVS
```

Comment:
Minor comment but this name could have been kept as CancerDisorderVS. It's implied as general and non-ambiguous if you have other value sets dedicated to PrimaryCancerCondition and SecondaryCancerCondition.
Also, we don't need to specify `Code` in the name of the value set. If so, for consistency, we could name all of our other value sets `AnyXCodeVS` if is supports more than one code system.

Reply:
OK, got rid of the funky name.